### PR TITLE
enhanced embeded design

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Description**

This PR 
- removed the provided by
- added support to get the styles to the Embed design to support both dark and light theme  

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
Looks great.

File: meshery-design-embed/public/embedded-design-embed1.js [hunk 1]
```diff
@@ -1622,7 +1618,7 @@ const addStyles = () => {
    #embedded-design-a3d3f26e-4366-44e6-b211-1ba4e1a3e644 .water-mark{
```
Comment (leecalcote) [inline meshery-design-embed/public/embedded-design-embed1.js:1618]: Why is there a specific design's UUID listed in this selector?

Comment (captain-Akshay) [reply meshery-design-embed/public/embedded-design-embed1.js:1618]: I think that's the default id given by @aabidsofi19

**Comment by leecalcote** [comment]
@captain-Akshay, is this a breaking change? We're using this package in a few places.

**Comment by captain-Akshay** [comment]
No, I don't think it will break anything. Though it might take default background colors for the pages where it is being used! @leecalcote

**Comment by leecalcote** [comment]
> No, I don't think it will break anything. Though it might take default background colors for the pages where it is being used! @leecalcote

@captain-Akshay, yes, this is what I mean by "break anything" (not the best description). Will you confirm which pages are impacted and are not impacted? Itemize each page in a list, please.

**Comment by leecalcote** [comment]
> No, I don't think it will break anything. Though it might take default background colors for the pages where it is being used! @leecalcote

Thank you for confirming. Do you see this as advantageous or unwanted behavior?

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
@captain-Akshay can you confirm that the embedded use here will continue with the desired background? - https://layer5.io/cloud-native-management/meshmap/design

**Comment by aabidsofi19** [comment]
@captain-Akshay  you have edited a sample embedded design  :smiley:  ,  the default styling and everthing else is done in meshmap

**Pull request merged**